Documentation/README_InteractiveElement.md

```diff
@@ -0,0 +1,91 @@
+# Interactive Element 
+
+An Interactive Element is the next generation of MRTK's Interactable.  An Interactive Element is an object that receives input and fires events depending on the type of input.  The type of input supported is defined in the form of states. 
+
+## Adding a Core States
```

Comment:
Should this be "Adding a Core State" or "Adding Core States" ?

Reply:
For the headlines, should we use sentence case?

Reply:
This should be Adding a Core State